apps/vscode/src/providers/format.ts

```diff
@@ -138,9 +138,13 @@ class FormatCellCommand implements Command {
       const edits = await formatActiveCell(editor, this.engine_);
       if (edits) {
         editor.edit((editBuilder) => {
-          edits.forEach((edit) => {
-            editBuilder.replace(edit.range, edit.newText);
-          });
+          // Sort edits by descending start position to avoid range shifting issues
```

Comment:
Unrelated but I thought we should make this robust to providers returning unsorted edits.